exporter/elasticsearchexporter/testdata/config.yaml

```diff
@@ -28,7 +27,6 @@ elasticsearch/log:
   endpoints: [http://localhost:9200]
   logs_index: my_log_index
   timeout: 2m
-  cloudid: TRNMxjXlNJEt
```

Comment:
I had to remove it because of the changes to `Config.Validate`:
 - this `cloudid` value is invalid
 - it's not permitted to have both `cloudid` and `endpoints` set

These would already have caused failures before, but only when creating the exporter, I just moved it to `Config.Validate` so the error occurs sooner.

There's a new valid entry called `elasticsearch/cloudid` which covers it.